src/functions/op.js

```diff
@@ -95,7 +95,8 @@ export const htmlBuilder: HtmlBuilderSupSub<"op"> = (grp, options) => {
             base = inner[0];
             base.classes[0] = "mop"; // replace old mclass
         } else {
-            base = buildCommon.makeSpan(["mop"], inner, options);
+            base = buildCommon.makeSpan(
+                ["mop"], buildCommon.tryCombineChars(inner), options);
```

Comment:
This is a new call to `tryCombineChars`, right? This suggests to me that it should be called in `buildExpression`, not anywhere else. For example, `\text{{hello}}` probably won't combine the characters in the current code, because of the ordgroup in the way, but would if we called it in `buildExpression`.

Perhaps it should only be called in text mode, though? How is that currently detected? (Vague worry that that's what the `italic > 0` check was for...)

Reply:
It is a new call.  It would be nice to handle cases like `\text{{hello}}`.  I tried collating `ordgroup`s but I had taken some of the checks out.  I should try again now that all of the checks are back in.  There are situations where I could see us not wanting to combine `ordgroups` though.  As an example: `{xyz}`... in this situation we might want to keep those characters separate in case someone wants to be able to highlight each variable on hover or click to select an individual variable in the term.  I think calling it for this particular case of `op` makes sense b/c it handles things like `sin`, `cos`, etc. and I think we should be treating those things as atomic units as opposed to separate letters.

Reply:
I get what you're saying... try to combine characters inside of ordgroup if we're in text mode.  I'll make an issue for it.  Having fewer `span`s should improve performance.  The fewer DOM nodes we have to create the better.